enterprise_gateway/services/processproxies/yarn.py

```diff
@@ -24,6 +24,16 @@
 max_poll_attempts = int(os.getenv('EG_MAX_POLL_ATTEMPTS', '10'))
 yarn_shutdown_wait_time = float(os.getenv('EG_YARN_SHUTDOWN_WAIT_TIME', '15.0'))
 
+# export EG_CERT_BUNDLE environment variable as string, path to CA Bundle
+# example:
+# export EG_CERT_BUNDLE="/var/lib/cloudera-scm-agent/agent-cert/cm-auto-global_cacerts.pem"
+cert_bundle = 'EG_CERT_BUNDLE'
+verify_default = True
+# cert_path: Boolean, defaults to `True`, that controls
+#            whether we verify the server's TLS certificate in yarn-api-client.
+#            Or a string, in which case it must be a path to a CA bundle to use.
+cert_path = os.getenv(cert_bundle, verify_default)
```

Comment:
Can we please collapse this down to the comments regarding cert_path and its simple initialization?  Users know how to set environment variables and we don't need the intermediate variables (`cert_ bundle` and `verify_default`).  Perhaps including a simple reference that the CA bundle is a pem file would be helpful.
```suggestion
# cert_path: Boolean, defaults to `True`, that controls
#            whether we verify the server's TLS certificate in yarn-api-client.
#            Or a string, in which case it must be a path to a CA bundle (pem file) to use.
cert_path = os.getenv('EG_CERT_BUNDLE', True)
```

Reply:
sure